app/rest/LastFm.js

```diff
@@ -12,7 +12,7 @@ function makeLastfmRequest (parameters) {
 }
 
 function searchTracks (terms, limit = 30) {
-  let parameters = 'track.search&track=' + encodeURI(terms);
+  let parameters = 'track.search&track=' + encodeURI(terms).replace('&', '');
```

Comment:
I agree with the issue, but would rather use `encodeURIComponent(terms)` in order to deal with other characters such as "#", "/" ,"." or ":"

Reply:
yeah, that´s a better solution... I´ve updated the file